ludwig/data/dataset/ray.py

```diff
@@ -82,20 +81,11 @@ def __init__(
         self._processed_data_fp = df if isinstance(df, str) else None
         self.auto_window = auto_window
 
-    def pipeline(
-        self,
-        shuffle: bool = True,
-        fully_executed: bool = True,
-        window_size_bytes: Optional[int] = None,
-        shuffle_seed: int = default_random_seed,
-    ) -> DatasetPipeline:
-        """
-        Args:
-            shuffle: If true, the entire dataset is shuffled in memory before batching.
-            fully_executed: If true, force full evaluation of the Ray Dataset by loading all blocks into memory.
-            window_size_bytes: If not None, windowing is enabled and this parameter specifies the window size in bytes
-                    for the dataset. If None and the dataset is large, set to the window size determined at init.
-        """
+    def get_window_size_bytes(self, window_size_bytes: Optional[int] = None):
```

Comment:
nit: We could add a type annotation for the return value, `def get_window_size_bytes(self, window_size_bytes: Optional[int] = None) -> int:`